src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -55,15 +59,15 @@ public ReportParser(Configuration configuration) {
             throw new ValidationException("No report file was added!");
         }
 
-        List<Feature> featureResults = new ArrayList<>();
-        for (int i = 0; i < jsonFiles.size(); i++) {
-            String jsonFile = jsonFiles.get(i);
-            Feature[] features = parseForFeature(jsonFile);
-            LOG.info("File '{}' contain {} features", jsonFile, features.length);
-            setMetadata(features, jsonFile, i);
-            featureResults.addAll(Arrays.asList(features));
+        List<Feature> featureResults = new ArrayList<>(jsonFiles.size() * 10);
```

Comment:
Remove magic number - should not have impact for performance as big as it was worth to provide specified number